beamer/config/commands.py

```diff
@@ -58,21 +58,21 @@ def _replay_event(w3: Web3, deployment: Deployment, config: Configuration, event
                 symbol = token.functions.symbol().call()
             except ContractLogicError:
                 # If token symbol is not available, use address instead
-                symbol = event.event_address
+                symbol = event.token_address
```

Comment:
Nitpick: would be nice to fold this fixup into the first commit, which introduced it.